src/status_im/contexts/wallet/events.cljs

```diff
@@ -786,7 +786,7 @@
  (fn [{:keys [db]}
       [{sent-transactions :sentTransactions
         send-details      :sendDetails}]]
-   (let [swap? (get-in db [:wallet :ui :swap])]
+   (let [swap? (get-in db db/swap)]
```

Comment:
Small question, do we always dissoc this state when it's not used? or is possible that it could be an empty map?

Reply:
better to have swap? as bool, so it should use `seq`

Reply:
@seanstrom, well we should dissoc it always but I can imagine that sometimes it is not the case due to bug.

@flexsurfer, sorry I didn't quite get what do you mean, could you please elaborate? 🙏